packages/fuels-macros/src/setup_program_test/code_gen.rs

```diff
@@ -197,7 +197,7 @@ impl Project {
 
     fn compile_file_path(&self, suffix: &str, description: &str) -> String {
         self.path
-            .join(["out/debug/", self.project_name(), suffix].concat())
+            .join(["out/release/", self.project_name(), suffix].concat())
```

Comment:
Everything besides `setup_program_test` is internal. This is a breaking change so should probably be "chore!:" in the PR title.

I seem to recall that we have a doc section for this macro. Wouldn't hurt to mention that it expects the forc project to be compiled in release mode in order to work.

That or better yet change the macro to have something like `setup_program_test!(Options(profile="release"), Deploy(contract="MyContract, wallet...`. But don't have to do it in this PR. Doc changes noting the release requirement and a task noting that it would be nice to have this be configurable is enough for now IMO.